dumper/app/src/main/resources/logback.xml

```diff
@@ -7,6 +7,13 @@
 		</encoder>
 	</appender>
 
+	<appender name="FILE" class="ch.qos.logback.core.FileAppender">
+		<file>dumper.log</file>
```

Comment:
I would suggest to include a timestamp in the filename of the progress log file, so that different executions don't overwrite the same file.

Reply:
Done